lineapy/graph_reader/node_collection.py

```diff
@@ -245,4 +256,12 @@ def get_input_parameters_block(self, indentation=4) -> str:
         else:
             input_parameters_codeblock = ""
 
+        if self.use_cache is not None:
+            logger.warning(
```

Comment:
Can we have a more descriptive warning here? Clarify this means that their input variables will have no effect because the "cache" overrides them, and give them mitigation options.